src/Microsoft.Azure.WebJobs.Host/JobHostQueuesConfiguration.cs

```diff
@@ -172,10 +175,28 @@ public TimeSpan VisibilityTimeout
         /// <see cref="QueueProcessor"/> instances that will be used to process messages.
         /// </summary>
         [CLSCompliant(false)]
-        public IQueueProcessorFactory QueueProcessorFactory 
-        { 
-            get; 
-            set; 
+        public IQueueProcessorFactory QueueProcessorFactory
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// Get's or sets the delete retry count
+        /// In some cases a queue may be unavailable temporarily. Retrying in some cases can
+        /// reduce the chance that a successfully processed message can return to the queue.
+        /// The default is 0
+        /// </summary>
+        public int DeleteRetryCount
```

Comment:
A default makes sense. I was thinking 5. Does it make sense to just put this in a private variable inside of QueueProcessor, or is there a better place to keep something like this?